driver/driver.go

```diff
@@ -0,0 +1,37 @@
+// Package driver defines the interface for storage driver implementations.
+// It provides a common interface for different storage backends like etcd and TCS.
+package driver
+
+import (
+	"context"
+
+	"github.com/tarantool/go-storage/driver/etcd"
```

Comment:
It seems that interface package shouldn't depends on its implementations.

Reply:
Yeah, to avoid cyclic dependencies in the future it is better to make the interface checks:

```
_ Driver = &tcs.DriverTCS{}
_ Driver = &etcd.DriverEtcd{}
```

In a test code in the driver_test.go or inside driver/* packages.

Reply:
fixed

Reply:
Will move checks to tests when they'll be implemented